src/spatial.jl

```diff
@@ -76,39 +76,43 @@ Ensure a coordinate is within the simulation box and return the coordinate.
 wrap_coords_vec(v, box_size) = wrap_coords.(v, box_size)
 
 """
-    velocity(mass, temperature; dims=3)
```

Comment:
Move this docstring to be directly above the function definition or it won't get attached. Also might be good to have both function signatures listed since `k` is optional.